web/src/context/ContextualLayers.svelte

```diff
@@ -14,17 +14,29 @@
 
 <Control defaultStyling={true}>
   <div
-    class="pico"
-    style:background="grey"
-    style="display: flex; flex-direction: column; max-height: 80vh; overflow: auto"
+    class="pico contextual-layers"
+    style="display: flex; flex-direction: column; max-height: 80vh; overflow: auto; border-radius: 5px;"
+    style:width={expand ? "300px" : "auto"}
   >
-    <button on:click={() => (expand = !expand)}>
-      <img src={icon} title="Layers" alt="Layers" />
+    <button
+      class="show-layers-button icon-btn {expand ? 'expanded' : ''}"
+      aria-label="Layers"
+      on:click={() => (expand = !expand)}
+    >
+      <div
+        style="display: flex; gap: 12px; color: #333; align-items: center; justify-content: center;"
+      >
+        <Layers color="#333" />
+        {#if expand}
+          Layers
+        {/if}
+      </div>
     </button>
 
     <div
-      style="display: flex; flex-direction: column"
-      style:visibility={expand ? "visible" : "collapse"}
+      style="flex-direction: column"
+      style:display={expand ? "flex" : "none"}
+      style:background-color="#515f7A"
```

Comment:
It's a bit hard to see some of the legend text against the grey; maybe we could try white foreground text?
![image](https://github.com/user-attachments/assets/d71f8ab9-7e7e-4517-a51e-5e6c0be1f571)